src/helperFunctions/hash.py

```diff
@@ -9,6 +11,12 @@
     complete_shutdown("Could not load ssdeep module. Install it via: BUILD_LIB=1 pip3 install ssdeep")
 
 
+try:
```

Comment:
lief installation should be added to the install script.
-> /src/bootstrap/bootstrap_backend.sh

Reply:
Please remove both try-except blocks around the imports.
We decided recently that those should be removed incrementally from the code base ; )